oci/spec_opts.go

```diff
@@ -273,6 +273,28 @@ func WithMounts(mounts []specs.Mount) SpecOpts {
 	}
 }
 
+// WithoutMounts removes mounts
+func WithoutMounts(dests ...string) SpecOpts {
+	return func(_ context.Context, _ Client, _ *containers.Container, s *Spec) error {
+		var (
+			mounts  []specs.Mount
+			current = s.Mounts
+		)
+	mLoop:
+		for _, m := range current {
+			mDestination := filepath.Clean(m.Destination)
```

Comment:
> --- FAIL: TestWithoutMounts (0.00s)
    spec_opts_test.go:650: expected [{Destination:/dst1 Type: Source:/src1 Options:[]} {Destination:/dst4 Type: Source:/src4 Options:[]}], got [{Destination:/dst1 Type: Source:/src1 Options:[]} {Destination:/dst2 Type: Source:/src2 Options:[]} {Destination:/dst3 Type: Source:/src3 Options:[]} {Destination:/dst4 Type: Source:/src4 Options:[]}]

Seems paths are formatted unexpectedly on windows?

`path.Clean` should be used, or `dest` should be `filepath.Clean`-ed as well.

Reply:
fixed